iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/process/last/LastQueryNode.java

```diff
@@ -46,31 +59,100 @@ public class LastQueryNode extends MultiChildProcessNode {
   // if children contains LastTransformNode, this variable is only used in distribute plan
   private boolean containsLastTransformNode;
 
+  private Map<IMeasurementSchema, Integer> measurementSchema2IdxMap;
```

Comment:
add some comments about this field to describe its life cycle, like when will this field be set to null